Documentation/HowToGuides/PassingAFloatValueFromAnInteractorToAnInteractable/README.md

```diff
@@ -0,0 +1,190 @@
+* `Vector2 Action` - This will be used if the input action from our controller is like a trackpad movement.
+
+In this example we are going to use the `Float Action` because we are going to use a trigger press.
+
+### Step 13
+
+We'll need to create a small script that allows us to change the opacity of our interactable GameObject to test that the trigger value from the controller passes over to the interactable.
+
+Create a new script by selecting `Main Menu -> Assets -> Create -> C# Script` in the Unity software and name it `OpacityChanger`.
+
+Copy and paste the below code into the newly created `OpacityChanger` script:
+
+```
+using UnityEngine;
+
+public class OpacityChanger : MonoBehaviour
+{
+    public GameObject meshToChange;
+
+    protected MeshRenderer materialRenderer;
+
+    protected void OnEnable()
+    {
+        materialRenderer = meshToChange.GetComponentInChildren<MeshRenderer>();
+    }
+
+    public void ChangeOpacity(float value)
+    {
+        Color currentColor = materialRenderer.material.color;
+        currentColor.a = 1f - value;
+        materialRenderer.material.color = currentColor;
+    }
+}
+```
+
+This simple script has a single method called `ChangeOpacity` which will take a float value that sets the opacity of the material that is used on the GameObject.
+
+Create a new GameObject as a child of the `OpacityChangingInteractable` and name it `OpacityChanger`. Then add the new `OpacityChanger` component to the GameObject.
+
+### Step 14
+
+We need to tell our `OpacityChanger` component what mesh to change. Drag and Drop the `Interactions.Interactable` GameObject into the `Mesh to Change` property on the `OpacityChanger` component.
+
+![Opacity Changer Script](assets/images/OpacityChangerScript.png)
+
+### Step 15
+
+Create a new material by selecting `Main Menu -> Assets -> Create -> Material` in the Unity software and name it `ChangingMaterial`. 
+
+Change the `Rendering Mode` property on the `ChangingMaterial` material to `Transparent`.
+
+> You could change the `Albedo` property on the `ChangingMaterial` material to another color using the color wheel.
+
+### Step 16
+
+Drag and drop the `ChangingMaterial` material from the Unity project assets onto the interactable cube in the Unity scene.
+
+![Drag And Drop Changing Material](assets/images/DragAndDropChangingMaterial.png)
+
+### Step 17
+
+Now to set up the `OpacityChanger` component to work when the trigger is pressed on either of the controllers.
+
+Expand the `OpacityChangingInteractable -> Interactions.ActionReceiver -> OutputActions` GameObject and select the nested `FloatAction` GameObject.
+
+Click the `+` symbol in the bottom right corner of the `Value Changed` event parameter in the `Float Action` component and then drag and drop the `OpacityChanger` GameObject into the box that appears and displays `None (Object)`.
+
+![Drag And Drop Opacity Changer Into Value Changed Parameter](assets/images/DragAndDropOpacityChangerIntoValueChangedParameter.png)
```

Comment:
Show the magenta box around the plus symbol being clicked

![image](https://user-images.githubusercontent.com/1029673/74438798-d83f2580-4e62-11ea-9aea-6fc6b36dbad4.png)